app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">KotlinMPP</string>
-</resources>
+</resources>
```

Comment:
You've deleted the new line at the end of this file, not a big deal but generally we aim to have new lines at the end of files  - it's a slightly old fashioned thing to do with the way that newlines / end of file is encoded, but more importantly github complains when there isn't one 😛